src/test/order_summary/OrderSummaryController.integration.test.ts

```diff
@@ -102,10 +110,12 @@ describe("OrderSummaryController", () => {
             const $ = cheerio.load(response.text);
             chai.expect(response.status).to.equal(404);
             chai.expect($(".govuk-heading-xl").text()).to.contain("Page not found");
+            chai.expect(getBasketStub).to.have.been.called;
```

Comment:
Did you try ` chai.expect(response.text).to.contain( "Basket (1)" );` here?

Reply:
Yes but complained